### PR TITLE
docs: add mapping entry for Ramda's `pluck`

### PR DESCRIPTION
**Pull request by eranhirsch**
`pluck` is our most popular missing search result after `throttle`.

### CODE REVIEW

File: docs/src/content/mapping/ramda/pluck.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+category: List
+remeda: map
+---
+
+_Not provided by Remeda._
+
+- `pluck(k, data)` is equivalent to `R.map(R.prop(k), data)`. `map` and `prop`
```
Comment (cjquines) [inline docs/src/content/mapping/ramda/pluck.md:8]: **question:** did you mean `R.map(data, R.prop(k))`?

**Review by cjquines — DISMISSED** [review]

File: docs/src/content/mapping/ramda/pluck.md (hunk 1 above)
Comment (eranhirsch) [reply docs/src/content/mapping/ramda/pluck.md:8]: This is from the ramda docs, they are only data-last

Comment (eranhirsch) [inline docs/src/content/mapping/ramda/pluck.md:8]:
```suggestion
- `pluck(k, data)` is equivalent to `map(data, prop(k))`. `map` and `prop`
```

**Review by eranhirsch — COMMENTED** [review]
better

File: docs/src/content/mapping/ramda/pluck.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+category: List
+remeda: map
+---
+
+_Not provided by Remeda._
+
+- `pluck(k, data)` is equivalent to `map(data, prop(k))`. `map` and `prop`
```
Comment (eranhirsch) [inline docs/src/content/mapping/ramda/pluck.md:8]:
```suggestion
- `pluck(data, k)` is equivalent to `map(data, prop(k))`. `map` and `prop`
```

**Review by eranhirsch — COMMENTED** [review]
improving clarity on data-first and data-last. Remeda users would expect data-first in examples.

File: docs/src/content/mapping/ramda/pluck.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+category: List
+remeda: map
+---
+
+_Not provided by Remeda._
+
+- `pluck(data, k)` is equivalent to `map(data, prop(k))`. `map` and `prop`
+  are available in Remeda so the same conversion is viable using these
+  functions.
```
Comment (eranhirsch) [inline docs/src/content/mapping/ramda/pluck.md:10]:
```suggestion
- `pluck(data, k)` is equivalent to `map(data, prop(k))`.
```

**Review by eranhirsch — COMMENTED** [review]
keeping it simple

File: docs/src/content/mapping/ramda/pluck.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+map(DATA, prop("val"));
+```
+
+### Objects
+
+```ts
+const DATA = { a: { val: "hello" }, b: { val: "world" } };
+
+// Ramda
+R.pluck("val", DATA); //=> { a: 3, b: 5 };
```
Comment (eranhirsch) [inline docs/src/content/mapping/ramda/pluck.md:30]:
```suggestion
R.pluck("val", DATA); //=> { a: "hello", b: "world" };
```

**Review by eranhirsch — COMMENTED** [review]
editing in comment :(

**Pull request merged**